drivingRange/NumToText/index.js

```diff
@@ -0,0 +1,90 @@
+const numberToText = [
+  "one",
+  "two",
+  "three",
+  "four",
+  "five",
+  "six",
+  "seven",
+  "eight",
+  "nine"
+];
+const teenToText = [
+  "ten",
+  "eleven",
+  "twelve",
+  "thirteen",
+  "fourteen",
+  "fifteen",
+  "sixteen",
+  "seventeen",
+  "eighteen",
+  "nineteen"
+];
+const writeNumber = number => {
+  var tempNumber = number;
+  if (number < 10) {
+    console.log(numberToText[number - 1]);
+  }
+  if (number > 9 && number < 20) {
+    console.log(teenToText[number - 10]);
+  }
+  if (number > 19 && number < 30) {
+    if (number == 20) {
+      console.log("Twenty");
+    } else {
+      console.log("Twenty-" + numberToText[number - 21]);
+    }
+  }
+  if (number > 29 && number < 40) {
+    if (number == 30) {
+      console.log("Thirty");
+    } else {
+      console.log("Thirty-" + numberToText[number - 31]);
```

Comment:
this looks like another pattern that can have a solution to address multiple cases/instances that can be resolved with a different approach. What is another way you can think of to handle these cases but would work for "fourty", "fifty", "sixty", and so forth?